bundles/org.openhab.binding.modbus.stiebeleltron/src/main/java/org/openhab/binding/modbus/stiebeleltron/internal/dto/EnergyBlock.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.modbus.stiebeleltron.internal.dto;
+
+/**
+ * Dto class for the Energy Block
+ *
+ * @author Paul Frank - Initial contribution
+ *
+ */
+public class EnergyBlock {
+
+    public int production_heat_today;
```

Comment:
Can you use camel case here?
```suggestion
    public int productionHeatToday;
```

Reply:
done